input/resources/OperationDefinition-bulk-submit-data.json

```diff
@@ -0,0 +1,75 @@
+{
+  "resourceType": "OperationDefinition",
+  "id": "bulk-submit-data",
```

Comment:
This is consistent with my suggestion to have .code element "bulk-submit-data".  The [.code](http://hl7.org/fhir/R4/operationdefinition-definitions.html#OperationDefinition.code) element determines what is used after the $ for the operation invocation.